### DISCUSSION TITLE
MegaTinyCore for servo on Arduino web editor?

### DISCUSSION THREAD

**Discussion by syedrazarizvi** [SpenceKonde/megaTinyCore#896] [Q&A]
I was able to download the megaTinyCore-master.zip and use the library (#include <Servo_megaTinyCore.h>) to run my servo with the Arduino desktop IDE 1.8.13. However, I get error messages when I try to import the zip file using the Arduino Web IDE. Am I doing something incorrectly or is it not compatible with the web IDE yet. Thanks!

**Comment by hmeijdam** [comment]
I would suspect it's the other way around.
The megaTinyCore-master.zip imports correctly into IDE 1.8.13 which proofs it works. If the Web Editor is unable to import that same zipfile, my conclusion would be that the web editor is not compatible with the regular IDE and/or MegaTinyCore.

**Reply by SpenceKonde** [reply]
I think the web editor is pretty damned flakey in general, I don't recommend it's use for that reason.
